developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -224,6 +226,26 @@ export class KmnCompiler implements UnicodeSetParser {
     for(let invalidVkey of invalidVkeys) {
       this.callbacks.reportMessage(CompilerMessages.Warn_InvalidVkeyInKvksFile({filename, invalidVkey}));
     }
+
+    if(displayMapFilename) {
+      // Remap using the osk-char-use-rewriter
+      let mapping: any;
+
+      displayMapFilename = this.callbacks.resolveFilename(kmnFilename, displayMapFilename);
+      try {
+        // Expected file format: displaymap.schema.json
+        // TODO: verify with schema
```

Comment:
I don't see a match for this among the PR's tracked items.  Is this being tracked somewhere/somehow?

Reply:
Good catch. Will add it here now so I don't forget -- it's a pretty simple addition.